tee-worker/ts-tests/vc.test.ts

```diff
@@ -4,19 +4,17 @@ import { requestVC, setUserShieldingKey } from './indirect_calls';
 import { Assertion } from './type-definitions';
 import { assert } from 'chai';
 import { u8aToHex } from '@polkadot/util';
-
 const assertion = <Assertion>{
     A1: 'A1',
     A2: ['A2', 'A2'],
     A3: ['A3', 'A3'],
     A4: [10, 'A4'],
-    A5: ['A5', 'A5'],
-    A6: 'A6',
+    // A5: ['A5', 'A5'],
+    // A6: 'A6',
     A7: [10, 10],
-    A8: [10],
-    A9: 'A9',
+    A8: 'A8',
+    // A9: 'A9',
```

Comment:
Any reasons to comment out these entries? Generally we shouldn't have commented code - either leave them or remove them - git has history.